pretest_clean.lua

```diff
@@ -25,9 +25,12 @@ local function clean()
     end):filter(function(name)
         -- skip internal spaces
         local first_char = string.sub(name, 1, 1)
-        return first_char ~= '_'
+        return (name ~= nil and first_char ~= '_')
     end):each(function(name)
-        box.space[name]:drop()
+        local status, err = pcall(function(name) box.space[name]:drop() end, name)
+        if not status then
```

Comment:
If I understand correctly, you are trying to filter all invalid values in the previous step, and a WARNING should not be printed at all.

Reply:
Right, not needed.